### PR TITLE
Add clarity in rerunning release drafter

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: guides/release-drafter.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org
 ### Other Recommendations
 
 - As the draft release description is generated on every push on `master`, don't change it manually until the final release publishment. Otherwise, your manual changes are going to be overwritten.
-- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again.
+- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again. If you're not sure where this action is: Click on action tab > in left sidebar `Release Drafter` > Click the latest action > right part of the screen click `re-run all jobs`.
```
Comment (bidoubiwa) [inline guides/release-drafter.md:34]:
```suggestion
- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again. If you're not sure where this action is: Click on the action tab > in the left sidebar `Release Drafter` > Click the latest action > right part of the screen click `re-run all jobs`. 
```

File: guides/release-drafter.md [hunk 2]
```diff
@@ -31,4 +31,4 @@ MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org
 ### Other Recommendations
 
 - As the draft release description is generated on every push on `master`, don't change it manually until the final release publishment. Otherwise, your manual changes are going to be overwritten.
-- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again.
+- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again. If you're not sure where this action is: Click on the action tab > in the left sidebar `Release Drafter` > Click the latest action > right part of the screen click `re-run all jobs`.
```
Comment (curquiza) [inline guides/release-drafter.md:34]:
```suggestion
- If you made any mistake (for example, the PR is already closed, but you forgot to add a label or you misnamed your PR), don't panic: change what you want in the closed PR and run the job again.

> 💡 How to re-run the "Release Drafter" job? 
> - Go to the `Action` tab.
> - Click on the left sidebar `Release Drafter`.
> - Select the latest action.
> - At the top right part of the screen click on `Re-run all jobs`.
```

**Review by curquiza — CHANGES_REQUESTED** [review]

**Review by curquiza — CHANGES_REQUESTED** [review]
Can you add the same paragraph in https://github.com/meilisearch/integration-guides/blob/master/guides/integration-tool-release.md please?

**Review by curquiza — APPROVED** [review]

File: guides/integration-tool-release.md [hunk 3]
```diff
@@ -30,4 +30,10 @@ Also, when writing the final changelogs, you might notice some inconsistencies,
 - Add the missing label (`breaking-change` or `skip-changelog`).
 - Re-run the last `Release Drafter` job in the `Actions` tab of the repository.
 
+> 💡 How to re-run the "Release Drafter" job? 
+> - Go to the `Action` tab.
+> - Click on the left sidebar `Release Drafter`.
+> - Select the latest action.
```
Comment (eskombro) [inline guides/integration-tool-release.md:36]: Is it "always" the latest action? Or could this be more specific?

Comment (bidoubiwa) [reply guides/integration-tool-release.md:36]: It's always the latest action!

Comment (curquiza) [reply guides/integration-tool-release.md:36]: I confirm

**Pull request merged**